server/test/account.test.js

```diff
@@ -0,0 +1,137 @@
+import { expect } from 'chai';
+import request from 'supertest';
+import app from '../src/app';
+import {
+  authUser,
+  newAccount,
+  emptyType,
+  invalidNewAccount,
+  newAccountTwo,
+  staffToken,
+  expiredToken,
+  fakeToken
+} from './helpers/fixtures';
+
+const URL = '/api/v1';
+
+let authToken;
+
+describe('Account Routes', () => {
+  before((done) => {
+    request(app)
+      .post(`${URL}/auth/signin`)
+      .send(authUser)
+      .end((err, res) => {
+        const { token } = res.body.data;
+        authToken = token;
+        if (err) return done(err);
+        done();
+      });
+  });
+
+  it('should create an account for a new user', (done) => {
+    request(app)
+      .post(`${URL}/accounts`)
+      .send(newAccount)
+      .set('Authorization', `Bearer ${authToken}`)
+      .expect(201)
+      .end((err, res) => {
+        expect(res.body).to.have.property('status').eql(201);
+        expect(res.body.data).to.have.property('id');
```

Comment:
Please check the value of the returned id

Reply:
done...